Using Bootstrap

rida-elbahtouri opened the pull request:
<h1>Using Bootstrap</h1>

#### What does this PR do?
created a Newsweek website clone based on Bootstrap 4

### in this project I :

- created a new HTML document.

- Think about all the elements on the page and how they are grouped together.

- Add Bootstrap to the project

- Layout the basic structure of the page using empty semantic HTML elements that are appropriately sized and which are positioned by using the appropriate Bootstrap classes.

- Set up the top title and navigation bar.

#### How should this be manually tested?
just clone the project and open the index.html file

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

rida-elbahtouri commented:
Thank @addod19 for your review, I have worked nav bar as required.

About the *Latest News Section* and *Featured Slideshows section* I am sorry there is no such section in the original page to clone. If there's one, please kindly leave me a screenshot.

The readme file live demo page seems to work very well on my end on the current feature branch: https://rawcdn.githack.com/rida-elbahtouri/-Bootstrap-project/86797a287a47ed33b08b2bdf224ea1f931e1331b/index.html
**other** than that I added  the navbar links and  the Date

lucasmazo32 commented on index.html:
```diff
@@ -0,0 +1,2294 @@
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <nav class="navbar fixed-top red-back">
```
- [ ] Kindly fix the navbar for large screens. At least in my end, it has differences. I also notice that the link provided on the PR is from a previous commit, kindly update it.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83579681-8c8eb580-a4ff-11ea-9c20-3ac11492b8d8.png) | ![image](https://user-images.githubusercontent.com/57414789/83579702-987a7780-a4ff-11ea-82c9-0b195c23cbad.png)

rida-elbahtouri commented:
fixed the navbar and updated the link:
https://rawcdn.githack.com/rida-elbahtouri/-Bootstrap-project/7d49d470a04f470c86503638e75027c5a80f564a/index.html

peterrobert commented:
**pull request under review by peter robert**

peterrobert commented on index.html:
```diff
@@ -0,0 +1,2320 @@
+            </div>
+          </div>
+        </div>
+        <div class="row ml-3">
+          <div class="col-lg-8 bg-light row be-fo">
+            <div class="col-sm-12 col-md-5 img-cont pr-0 position-relative">
+              <div class="position-absolute subscribe-img-2">
+                <img
+                  src="assets/img/may-29-june-06-2020-cover.jpg"
+                  alt="may-29-june-06-2020-cover"
+                />
+              </div>
+              <div class="position-absolute subscribe-img-1">
+                <img
+                  src="assets/img/may-22-2020-cover.jpg"
+                  alt="may-22-2020-cover"
+                />
+              </div>
+            </div>
+            <div class="col-sm-12 pl-0 col-md-7 mt-5 pr-2 pt-2">
+              <h5 class="font-weight-bold">
+                In-depth Stories You Don't See Elsewhere About Topics You Want
+                to Read!
+              </h5>
+              <p>
+                All Access + Weekly Delivery. Choose A Membership That's Perfect
+                for You!
+              </p>
+              <button
+                class="btn red-back px-2 py-1 text-uppercase text-light font-weight-bold"
+              >
+                Subscribe >
+              </button>
+            </div>
```
**[REQUIRED]** kindly fix the text in this block. It is below the image and hence difficult to read on mobile and medium view
![snapshot-1591175528832](https://user-images.githubusercontent.com/32361963/83619147-1b4d1200-a594-11ea-9efc-b32612e79811.jpg)
